modules/table/src/main/java/org/apache/ignite/internal/table/AbstractTableView.java

```diff
@@ -135,7 +135,7 @@ protected final <T> CompletableFuture<T> withSchemaSync(@Nullable Transaction tx
 
     private <T> CompletableFuture<T> withSchemaSync(@Nullable Transaction tx, @Nullable Integer previousSchemaVersion, KvAction<T> action) {
         CompletableFuture<Integer> schemaVersionFuture = tx == null
-                ? schemaVersions.schemaVersionAtNow(tbl.tableId())
+                ? schemaVersions.weakSchemaVersionAtNow(tbl.tableId())
```

Comment:
Does it make sense to add a comment explaining the usage of the weak version? Like 'We use the weak version of the method to avoid ticking the clock (to decrease contention); it might return a stale schema, but the retry mechanism implemented below allows us to be safe in this case'

Reply:
Added the comment.